api/SVGFECompositeElement.json

```diff
@@ -51,40 +51,52 @@
         "__compat": {
           "support": {
             "chrome": {
-              "version_added": "5"
+              "version_added": "5",
+              "notes": "BackgroundImage/BackgroundAlpha are not supported (see <a href='https://crbug.com/137230'>Chromium bug 137230</a>)."
```

Comment:
FYI added all these notes to mirror `feComposite` (which this API does).

Reply:
Essentially yes. You can specify those of course (they are "valid" options) but the only thing they they work on is pre-Blink-Edge. So "no current browser". 

My read of the defect reports is because this is very hard to do, not because it isn't a good idea. Even the example in the SVG spec uses these. I have just removed that example from the docs. Also yes, that some browsers won't ever fix this.

But my concern is that without the note everyone will think all the spec values for in1 and in2 will work. I think you're saying "that doesn't matter". I will take your direction and update appropriately :-)

Reply:
1. To clear up a point of confusion: reflected attributes set and get the source element's attributes. For example, given page source `<feComposite in1="somevalue">`, `SVGFECompositeElement.in1` returns `"somevalue"` and `SVGFECompositeElement.in1 = "NEW"` sets the element to `<feComposite in1="NEW">`. In terms of behavior, reflected attributes provide access to underlying element but don't really add anything more.

   Consequently, to avoid duplication, we document support for attribute values on the element itself, not the interface to that element. We figure that people don't really learn about, for example, the attributes of HTML elements by reading the `HTML*Element` interface docs.

2. > my concern is that without the note everyone will think all the spec values for in1 and in2 will work
    
    I didn't realize the spec used these (unsupported) values as examples and I appreciate that you're trying to guard against misunderstanding here. In that case, I think the right way to handle this is to have subfeatures for `in1` and `in2` (on the `svg.elements.feComposite.{in1,in2}` features) that represent each of the values for it and provide complete compat data for each, instead of notes. I think that gives readers a clearer idea of what is and is not supported.

    There's one additional bit of nuance here, however: because spec authors write ahead of implementation, there are often features which are in specs and never implemented. BCD can be a source of additional confusion, as folks get the idea that a feature in BCD means that it's a live concern. This is why we avoid all `false` features, where possible.
    
    In this case, the `BackgroundImage` and `BackgroundAlpha` values were once supported by at least one browser, so that deserves an entry.

3. I was trying to avoid repetitious notes. Repetitious notes are hard to follow. When every entry has a note and the text is very similar between them, then it's hard to know what's actually noteworthy. That's why I was leaning toward writing notes for the exceptional, not usual, case.

Reply:
Two additional thoughts here:

- I missed that IE supported BackgroundImage and BackgroundAlpha as well. I think my general point still stands, however, that these are interesting because they're different from the others.
- I said that we should "have subfeatures for `in1` and `in2` (on the `svg.elements.feComposite.{in1,in2}` features) that represent each of the values". I am not asking you to add features for every value yourself as part of this PR (that's optional). I just meant that `svg.elements.feComposite.{in1,in2}.BackgroundImage` and `svg.elements.feComposite.{in1,in2}.BackgroundAlpha` would be germane here.

Reply:
@ddbeck You didn't miss anything about IE. I'm an idiot - those notes were supposed to be on Edge. Fixed. 

But my feeling is that this is probably worth merging now because it is "not wrong" and adds a lot of corrected information about supported versions. 

There are ongoing discussions like https://github.com/mdn/browser-compat-data/pull/9074#discussion_r579043292 and this one on possibly adding subfeatures, but I'd happily do those as a post process. Up to you.